Bucketeer.xcodeproj/project.pbxproj

```diff
@@ -1619,11 +1631,11 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = VPN42CNA7H;
+				DEVELOPMENT_TEAM = 4SVK7N5V6K;
 				INFOPLIST_FILE = Example/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = io.bucketeer.sdk.ios.example;
+				PRODUCT_BUNDLE_IDENTIFIER = io.bucketeer.sdk.ios.example2;
```

Comment:
I will revert this bundle later.
To enable background mode, the bundle needs to register under a developer account.
`io.bucketeer.sdk.ios.example` was register to other account before.
I needed it to change it a bit to run the example on my phone.